inc/sainexthopgroup.h

```diff
@@ -133,6 +202,15 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_COUNTER_ID,
 
+    /**
+     * @brief Maximum number of sessions that the group can handle
+     *
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @condition SAI_NEXT_HOP_GROUP_ATTR_TYPE == SAI_NEXT_HOP_GROUP_TYPE_L4_SESSION_CONSISTENT
+     */
+    SAI_NEXT_HOP_GROUP_ATTR_MAX_SESSIONS,
+
```

Comment:
Number of sessions doesn't work in case of hashing IMO.
How about:
/**
     * @brief Maximum number of entries that the group can handle.
     * If each entry corresponds to a session, this is the maximal number of sessions.
     * If each entry corresponds to a hash bucket, this is the maximal number of buckets.
     *
     * @type sai_uint32_t
     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
     * @condition SAI_NEXT_HOP_GROUP_ATTR_TYPE == SAI_NEXT_HOP_GROUP_TYPE_L4_SESSION_CONSISTENT
     */
    SAI_NEXT_HOP_GROUP_ATTR_MAX_ENTRIES,